ldap/servers/slapd/conntable.c

```diff
@@ -154,7 +154,7 @@ connection_table_new(int table_size)
         /* We rely on the fact that we called calloc, which zeros the block, so we don't
         * init any structure element unless a zero value is troublesome later
         */
-        for (i = 0; i < ct->list_size; i++) {
+        for (i = 1; i < ct->list_size; i++) {
```

Comment:
Isn't the first element of the conntable  a linked list header/pointer? I think that's why this was set to 1 here because the 0'th slot was reserved for LL chasing.

Reply:
@Firstyear 
It is, I cant believe I missed that one :(

Reply:
It's really not obvious, even I overlooked it when I first worked on that part of the code.

So sadly you'll need to revert this change :(

Reply:
This commit broke some of the CI tests, it has been fixed in https://github.com/389ds/389-ds-base/pull/5917